src/common/BaseAnnotator.ts

```diff
@@ -121,6 +121,22 @@ export default class BaseAnnotator {
 
     setVisibility = (visibility: boolean): void => {
         this.store.dispatch(store.setVisibilityAction(visibility));
+
+        this.setLayerVisibility(visibility);
+    };
+
+    setLayerVisibility = (visibility: boolean): void => {
+        const layerList = document.querySelectorAll('.ba-Layer');
+        if (!layerList.length) {
+            return;
+        }
+        layerList.forEach(element => {
```

Comment:
We should avoid re-rendering every page. Can we apply a class to the `container` element?